vars/list_preseeds.yml

```diff
@@ -1,14 +1,14 @@
 ---
 list_preseeds:
     - name: "Debian"
-      description: "Debian"
+      description: "Debian with automated partitioning"
       type: "preseed"
       path: "./files/preseeds/Debian.preseed"
-    - name: "CentOS"
-      description: "CentOS"
+    - name: "CentOS.Upstream"
```

Comment:
Names can remain with spaces, it's file names that shouldn't

Reply:
Everything has a dot now !